### DISCUSSION TITLE
Jerry Mayor as Minister in API

### DISCUSSION THREAD

**Discussion by j10a1n15** [HypixelDev/PublicAPI#671] [Ideas]
This is a suggestion for both ingame and for the API, it fits more into the API though.

Since Special mayors cant have ministers active, the Jerry Perkpocalypse mayor could be a temporary minister instead.
To keep showing the Jerry  perkpocalypse mayor in the calendar, you can also add another head next to the current mayor like it is in SkyHanni right now:
![image](https://github.com/user-attachments/assets/65fb0a9f-65c9-4ea5-a1b7-081a27a37c8c)

This would also finally add the Perkpocalypse mayor into the API

**Comment by DuckySoLucky** [comment]
> This would also finally add the Perkpocalypse mayor into the API

Unfortunately not possible, we've already asked Connor (And I don't believe that they would rewrite the entire system only because of this)

![image](https://github.com/user-attachments/assets/1ff39e39-3c55-4123-8a98-25675e7d28de)

**Comment by j10a1n15** [comment]
> Unfortunately not possible, we've already asked Connor (And I don't believe that they would rewrite the entire system only because of this)

I dont want the full order, I would already be happy with just the current mayor
While im not sure how their code handles that, I might have a feeling that since it already shows the current perkpocalypse mayor, that one is exposed easier? Not sure
